source/components/molecules/HelpButton/HelpButton.js

```diff
@@ -68,7 +69,7 @@ const LinkButton = styled(Button)`
 
 const HelpButton = (props) => {
   const { text, size, heading, tagline, url } = props;
-  const [modalVisible, setModalVisible] = useState(false);
+  const [visible, toggleModal] = useModal();
```

Comment:
what is visible?

Reply:
Changed visible state to => isModalVisible